SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizer.cc

```diff
@@ -92,10 +97,13 @@ namespace cms {
         pSetupToken_(iC.esConsumes()) {
     edm::LogInfo("PixelDigitizer ") << "Enter the Pixel Digitizer";
 
+    edm::LogInfo("PixelDigitizer ") << " applyLateReweighting_ " << applyLateReweighting_;
+
     const std::string alias("simSiPixelDigis");
 
     producesCollector.produces<edm::DetSetVector<PixelDigi> >().setBranchAlias(alias);
     producesCollector.produces<edm::DetSetVector<PixelDigiSimLink> >().setBranchAlias(alias + "siPixelDigiSimLink");
+    producesCollector.produces<edm::DetSetVector<PixelSimHitExtraInfo> >().setBranchAlias(alias + "siPixelExtraSimHit");
```

Comment:
Would it make sense to have this under `if(applyLateReweighting_)`? This is to avoid producing and putting in the event an empty collection.

Reply:
yes, I manage to make it work. the logic need `if (store_SimHitEntryExitPoints_)` in place of `if(applyLateReweighting_)` and also to play with the aliases.py file ([for standard cases without the extra collection](https://github.com/carolinecollard/cmssw/blob/additionalDigiInfo_12_4_X/SimGeneral/MixingModule/python/aliases_cfi.py#L34:L37) and ([for Late CR in the runDependent scenario](https://github.com/carolinecollard/cmssw/blob/additionalDigiInfo_12_4_X/SimGeneral/MixingModule/python/aliases_cfi.py#L98:L99).